engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -551,8 +551,8 @@ public void dispose() {
         renderableWorld.dispose();
         worldProvider.dispose();
         // TODO: Shift this to a better place, after a RenderGraph class has been implemented.
-        // TODO: Call ResetDefaultInstance() for every StateChange after it is a method of StateChange.
-        SetViewportToSizeOf.ResetDefaultInstance();
+        // TODO: Call disposeDefaultInstance() for every StateChange after it is a method of StateChange.
```

Comment:
Fair enough, I'll drop the comment (and the idea) for the time being, and will try again when I am more convinced of the idea myself xD